xija/component/heat.py

```diff
@@ -400,13 +408,58 @@ def __init__(self, model, node,
         self.n_mvals = 1
         self.add_par('k', k, min=0.0, max=2.0)
 
+    def calc_earth_vis_from_grid(self, ephems, q_atts):
+        import astropy_healpix
```

Comment:
With this as a non-top import, should astropy_healpix be added as a xija package (or conda) requirement or not?